lua/autorun/server/sv_cfc_explosion_antispam_init.lua

```diff
@@ -0,0 +1,76 @@
+CFC_ExplosionAntispam = CFC_ExplosionAntispam or {}
+
+CFC_ExplosionAntispam.NEAR_DIST = math.pow( 250, 2 ) --How close explosive damage events have to be for limitations to apply
+CFC_ExplosionAntispam.MAX_NEAR = 10 --The max number of explosive damage events allowed in an area
+CFC_ExplosionAntispam.EXPLOSION_TIMEOUT = 1 --How long explosive damage events stay logged for
+CFC_ExplosionAntispam.BARREL_TIMOUT = 2 --Base timeout duration for explosive barrels
+CFC_ExplosionAntispam.BARREL_TIMOUT_OFFSET_MAX = 1 --Adds random extra timeout duration from 0 to this value to barrels to look more natural
+CFC_ExplosionAntispam.STOP_ALL_DAMAGE_TIMEOUT = 10 --Default time that stopAllDamage() lasts for
```

Comment:
I'd prefer these on a subtable:
```suggestion
CFCExplosionAntispam.Settings = {
    NEAR_DIST = math.pow( 250, 2 ), -- How close explosive damage events have to be for limitations to apply
    MAX_NEAR = 10, -- The max number of explosive damage events allowed in an area
    EXPLOSION_TIMEOUT = 1, -- How long explosive damage events stay logged for
    BARREL_TIMOUT = 2, -- Base timeout duration for explosive barrels
    BARREL_TIMOUT_OFFSET_MAX = 1, -- Adds random extra timeout duration from 0 to this value to barrels to look more natural
    STOP_ALL_DAMAGE_TIMEOUT = 10 -- Default time that stopAllDamage() lasts for
}
```